testscommon/testdata/stateChanges.json

```diff
@@ -0,0 +1,292 @@
+{
+  "shardID": 4294967295,
+  "blockData": {
+    "shardID": 4294967295,
+    "headerBytes": "CAEYHiCu0MW3BipPEiAYREYUy0aMcT1PKB5VFme7cboLZ7zPJCeXpyXlw8GOciIgjKOogIKxIEr8aPQUwGWwG6TWfjYEYbM5Ywv7StyubacqAQFiAgAAcgIAADowvZq1GpGrIzIRC9aUskUEGdfIxBhphB2yiTU5jME6eGrJ253zsG2DOL9f0qEXuLsQQjB8ZYtDmqREJouNIPCG7KKlCHYp2i9o8EdZNqdNvagV7dz/lomm5aj5porltRoelYJKAQFSIFS6tA9tdZKyRqNWupjw13/Qpbtg5bKfQbfW+jdoLWi7WiBZIyxvcVkjg3XEd3xRXXYkU3gKpYs3HcK5ZDQ5D7SQhWIw9c+u9YCRtfG+hT+iLrfJhF3u8QXgfzE4rnsWIIOtnNhpUQLu37fBTIg7NG0AyQ+KaiBZIyxvcVkjg3XEd3xRXXYkU3gKpYs3HcK5ZDQ5D7SQhXIgT79mjjyBuqvBU9XvDViKqdCIMImy6gmA2m3AILPtA0aKASAOV1HAJuVDsuirLrBgmdqh0eXfR3ePd4f6q0XN8S/jqJIBFBISCgEAEgEAGgEAIgEAKgEAMgEAmgENbG9jYWwtdGVzdG5ldKIBB2RlZmF1bHSqAQIAALIBAgAAugECAADCAQIAAA==",
+    "headerType": "MetaBlock",
+    "headerHash": "fx94JbIPrwcJZ/lDI1X1goBjzH2G1vZk+4gwchUysis=",
+    "body": {}
+  },
+  "transactionPool": {},
+  "headerGasConsumption": {
+    "gasProvided": 0,
+    "gasRefunded": 0,
+    "gasPenalized": 0,
+    "maxGasPerBlock": 15000000000
+  },
+  "notarizedHeadersHashes": [
+    "18444614cb468c713d4f281e551667bb71ba0b67bccf242797a725e5c3c18e72"
+  ],
+  "numberOfShards": 1,
+  "signersIndexes": [
+    0
+  ],
+  "highestFinalBlockNonce": 0,
+  "stateChanges": {
+    "M�hᎧ\u001fG\u0006�f�鼖+:bg⾩�1,�\u0015^��\t\u0018": {
```

Comment:
can we have the hashes encoded as hex or erd? and decode it in tests if needed

Reply:
the state changes were taken with the state changes collector available in the node. This change should be relflected there as well.

Reply:
changed.